internal/sqladapter/testing/adapter.go.tpl

```diff
@@ -1092,15 +1092,14 @@ func TestBatchInsert(t *testing.T) {
 
 		go func() {
 			for i := 0; i < totalItems; i++ {
-				batch.Values(fmt.Sprintf("artist-%d", i))
+				batch.Values <- fmt.Sprintf("artist-%d", i)
```

Comment:
The nomral API for inserter.Values() currently accepts a variadic, so we usually do:

```
inserter.Values(1, 2, 3)
```

This would only insert one element and I am not sure if you can insert many without something like:

```
batch.Values <- []interface{1, 2, 3}
```

Which would work but seems a bit awkward.